package.json

```diff
@@ -46,7 +47,7 @@
     "url": "git+https://github.com/transcovo/express-middlewares.git"
   },
   "engines": {
-    "node": "8.10.0",
+    "node": ">=6.0.0",
     "npm": ">=2.14.7"
```

Comment:
You should remove this and let the npm version depend on the node version installed

Reply:
ok